ui/raidboss/data/04-sb/dungeon/fractal_continuum_hard.txt

```diff
@@ -37,7 +37,7 @@ hideall "--sync--"
 
 # -ii 2798
 
-# Unfortunately, we have no choice but to use game log lines here.
+# Unfortunately, we have no choice but to use BattleTalk2 lines here.
 # The statues the boss tethers to are useless for timeline syncs,
 # and the boss gains no buffs for each form.
```

Comment:
I'll see if I can check this one to see if it's the case.

Reply:
This is my work and it's very likely this isn't true anymore. At the time, the statues were spawned in at the time the boss room was opened, and so there wasn't any reliable way to determine what we were tethered to. I *think* it might be possible to use the tethers in combination with BNPCID or NPCID now, instead of using Battletalk.

Reply:
I'll re-run it as well and see if there are alternatives, but as a general philosophy with this eliminate-GameLog-triggers effort, I'm admittedly trying to preference "replacement" over "rework" where possible, the goal being to reduce the support issues that arise from having chat log hidden for privacy. Since triggering off of something other than the messages (whatever their source) can require adjustments to the timeline, I'm sort of trying to avoid that option unless necessary.

Reply:
> This is my work and it's very likely this isn't true anymore. At the time, the statues were spawned in at the time the boss room was opened, and so there wasn't any reliable way to determine what we were tethered to. I _think_ it might be possible to use the tethers in combination with BNPCID or NPCID now, instead of using Battletalk.

I looked at it a little more.  The boss does tether to each statue with an 0x23 line, but the tether ids are the same, so all we have to differentiate them on the 0x23 line is combatant ids.  Unfortunately, all of the combatants have the same bNpcNameId and name; the only thing distinguishing them is their x/y position.  Assuming each of the 3 positions always corresponds to the same mode (e.g. Sephirot->Sophia->Zurvan, left to right) -- and I didn't actually run it multiple times to verify that -- we'd have to determine which combatant id corresponds to which primal based on the combatant's position, and then jump/sync based on which combatant id Ultima Warrior tethers to.  That wouldn't be terribly difficult for a set of triggers, but I'm actually not sure how we'd dynamically handle that for a timeline entry.  (Also, the tethers come a bit after the BattleTalk2 packet, so we'd be looking at timeline rework anyway).

Given all that -- and the fact this is content from 2 expacs ago -- I think I'm leaning toward leaving this as BattleTalk2